src/constants/routes.test.ts

```diff
@@ -7,8 +7,31 @@ import {
   getPatchRoute,
   getTaskHistoryRoute,
   getVariantHistoryRoute,
+  getProjectPatchesRoute,
+  getCommitQueueRoute,
+  getCommitsRoute,
 } from "./routes";
-
+const identifier = "!?identifier@";
+const escapedIdentifier = "!%3Fidentifier%40";
+describe("getProjectPatchesRoute", () => {
```

Comment:
nit. add some spacing and rename the variable so its more clear what it is.

```suggestion

const identifierWithSpecialCharacters = "!?identifier@";
const escapedIdentifier = "!%3Fidentifier%40";

describe("getProjectPatchesRoute", () => {
```